dashboard/src/t5gweb/utils.py

```diff
@@ -190,6 +190,8 @@ def set_cfg():
         cfg["sla_settings"] = json.loads(os.environ.get("sla_settings"))
     # sso
     cfg["rbac"] = os.environ.get("rbac").split(",") if os.environ.get("rbac") else []
+    cfg["max_to_create"] = os.environ.get("max_to_create")
+    cfg["alert_email"] = os.environ.get("alert_email")
```

Comment:
`alert_email` is set to `cfg['alert_to']` above, maybe we should unify these two